src/operations.ts

```diff
@@ -73,6 +80,7 @@ function resolveRequestBody(args: any[]) {
             default: arg.defaultValue,
             ...resolveFieldType(arg.type),
             deprecated: !!arg.deprecationReason,
+            nullable: isNullableType(arg),
```

Comment:
This property should be added as optional something like below
...(isNullableType(arg) && { nullable: isNullableType(arg) }),

Reply:
yes will update this